src/daft-session/src/python.rs

```diff
@@ -93,11 +102,21 @@ impl PySession {
     pub fn set_namespace(&self, ident: Option<&PyIdentifier>) -> PyResult<()> {
         Ok(self.0.set_namespace(ident.map(|i| i.as_ref()))?)
     }
-}
 
-impl From<&PySession> for Session {
-    fn from(sess: &PySession) -> Self {
-        sess.0.clone()
```

Comment:
Why was this removed? unused now?

Reply:
see https://github.com/Eventual-Inc/Daft/pull/4174#discussion_r2045704196